ui/flow.js

```diff
@@ -46,18 +47,38 @@ async function loadSteps(steps) {
         mermaidDiv.appendChild(mermaidPane);
       }
     }
-    // yamlDiv.innerHTML =
-    //   '<pre class="yaml-content">' +
-    //   (step?.api === "form" ? step.example.value : JSON.stringify(step.example.value, null, 2)) +
-    //   "</pre>";
-    yamlDiv.innerHTML = step?.api === "form" ? '<div>'+'<pre class="yaml-content">'+'<xmp>'+step.example.value+'</xmp>'+'</pre>'+'<div class="flow-forms">'+step.example.value+'</div>'+'</div>'
-      :'<pre class="yaml-content">' +
-       JSON.stringify(step.example.value, null, 2) +
+
+  
+    const copyButton = document.createElement("div");
+    copyButton.textContent = "";
+    copyButton.classList.add("copy-code-button");
+    copyButton.style.backgroundImage = 'url("../icons/icon-copy.png")';
+
+    copyButton.addEventListener("click", function (event) {
+      event.preventDefault();
+      const textArea = document.createElement("textarea");
+      textArea.value = JSON.stringify(step.example.value, null, 2);
+      document.body.appendChild(textArea);
+      textArea.select();
+      document.execCommand("copy");
+      document.body.removeChild(textArea);
+      copyButton.style.backgroundImage = 'url("../icons/icon-tick.png")';
```

Comment:
path for icons will not work here, as this files is being consumed at root level, please change it to `'url("icons/icon-tick.png")'`